DependencyInjection/Configuration.php

```diff
@@ -107,7 +107,11 @@ public function getConfigTreeBuilder()
                                 return $v;
                             })
                         ->end()
-                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))->end()
+                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))
+                    ->end()
+                    ->scalarNode('service_naming_strategy')
```

Comment:
Sure, but without a compiler pass override is the order of declaration+override guaranteed in any way?